platform/server/elasticsearch/__tests__/call_api.test.ts

```diff
@@ -0,0 +1,76 @@
+import { Client } from 'elasticsearch';
+import { callAPI } from '../call_api';
+
+test('should call the api when it exists, with the right context and params', async () => {
+  let apiContext;
```

Comment:
We need to replace these with `let apiContext: any;` (this will be clear when we merge https://github.com/elastic/kibana/pull/15576)